tests/rptest/tests/rpk_group_test.py

```diff
@@ -0,0 +1,151 @@
+# Copyright 2022 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from ducktape.utils.util import wait_until
+from rptest.clients.rpk import RpkTool
+from rptest.clients.types import TopicSpec
+from rptest.services.cluster import cluster
+from rptest.services.rpk_consumer import RpkConsumer
+from rptest.services.rpk_producer import RpkProducer
+from rptest.tests.redpanda_test import RedpandaTest
+
+
+class RpkGroupCommandsTest(RedpandaTest):
+    def __init__(self, test_ctx, *args, **kwargs):
+        self._ctx = test_ctx
+        super(RpkGroupCommandsTest, self).__init__(
+            test_ctx,
+            *args,
+            **kwargs,
+        )
+
+    def _rpk_produce_to_topic(self, topic, n=10):
+        self.rpk = RpkTool(self.redpanda)
+        self.rpk.create_topic(topic)
+
+        for p in range(20):
+            self.rpk.produce(topic, f"k-{p}", f"v-{p}")
+
+    def setup_producer(self, p_cnt, topic_name):
+        # create topic
+        self.topic_spec = TopicSpec(partition_count=p_cnt, name=topic_name)
+        self.client().create_topic(specs=self.topic_spec)
+        # produce some messages to the topic
+        self.producer = RpkProducer(self._ctx, self.redpanda,
+                                    self.topic_spec.name, 128, 500)
+        self.producer.start()
+
+    @cluster(num_nodes=3)
+    def test_group_list_and_delete(self):
+        """
+        Simple listing and deleting one of the groups.
+        """
+        topic = "test_group"
+        self._rpk_produce_to_topic(topic, 20)
+
+        group_1 = "test-g1"
+        self.rpk.consume(topic, group=group_1, n=10)
+
+        group_2 = "test-g2"
+        self.rpk.consume(topic, group=group_2, n=10)
```

Comment:
Perhaps worth commenting that groups show up until they're deleted or until all offsets expire (I think those are the two cases).